packages/salesforce/src/Adaptor.js

```diff
@@ -308,10 +312,18 @@ async function pollJobResult(conn, job, pollInterval, pollTimeout) {
 export function bulkQuery(qs, options, callback) {
   return async state => {
     const { connection } = state;
-    const [resolvedQs, { pollTimeout = 90000, pollInterval = 3000 }] =
-      newExpandReferences(state, qs, options);
+    const [resolvedQs, resolvedOptions] = newExpandReferences(
+      state,
+      qs,
+      options
+    );
     const apiVersion = connection.version;
 
+    const { pollTimeout, pollInterval } = {
```

Comment:
Hey @mtuchi what's up with this change? I much prefer the old syntax?

Reply:
It's buggy, I ran into some problem before. So i think i shouldn't make a change in this PR for now

Reply:
hmm :thinking: 

OK, at least it's consistent with main, so I'll let this pass

Just version bumps next please! You may need to merge or rebase against main after the surveycto stuff